plio/models.py

```diff
@@ -95,6 +95,8 @@ class Question(SafeDeleteModel):
     correct_answer = models.TextField(null=True)
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+    has_char_limit = models.BooleanField(default=False)
+    max_char_limit = models.FloatField(null=True)
```

Comment:
can't we just use `max_char_limit`?
As it's null by default, we can assume that this question doesn't have any `max_char_limit` set
That way we won't need another field in our model